omniplanner_ros/src/omniplanner_ros/omniplanner_node.py

```diff
@@ -194,6 +204,10 @@ def get_spot_pose(self):
     def register_plugin(self, name, plugin):
         self.get_logger().info(f"Registering subscription plugin {name}")
         msg_type, topic, callback = plugin.get_plan_callback()
+        try:
```

Comment:
I want to get rid of this try block before we eventually merge, but this is great for initial testing. I will go back and add the `get_plugin_feedback` functions for the other planners if / when we confirm we like this feedback architecture